test/test.js

```diff
@@ -144,6 +162,20 @@ describe( "Localization", function() {
         );
     });
 
+    it( "uses the default locale if language code query parameter is not in the list", function( done ) {
```

Comment:
I think the best way is to use the default locale if the query language does not exists. If not, why have a default locale ? 
We could know if we return a 404 or not if the `defaultLocale` parameter exists